src/crimson/os/seastore/cache.cc

```diff
@@ -501,6 +506,16 @@ Cache::get_extent_ertr::future<CachedExtentRef> Cache::get_extent_by_type(
       ).safe_then([](auto extent) {
         return CachedExtentRef(extent.detach(), false /* add_ref */);
       });
+    case extent_types_t::OMAP_INNER:
+      return get_extent<omap_manager::OMapInnerNode>(offset, length
+      ).safe_then([](auto extent) {
+        return CachedExtentRef(extent.detach(), false /* add_ref */);
```

Comment:
For a future change: replace the 'bool' parameter (that required a comment here) with a typed interface

Reply:
this is just follow the others currently.  maybe we modify it later.